frontend/src/components/dashboard/varianceHomePage/VarianceHomePage.js

```diff
@@ -280,15 +303,14 @@ export class VarianceHomePage extends Component {
           <h1>Browse Variances</h1>
         </div>
         <div className="landing-page__content">
-          <AuthorizationWrapper permission={Permission.IN_TESTING}>
```

Comment:
Does this PR include the sorting? We shouldn't put this in prod until the table has sorting

Reply:
done